extensions/smoke-test/README.md

```diff
@@ -0,0 +1,3 @@
+# smoke test
+
+A document which exercises all of Connect's supported runtimes.
```

Comment:
As this is written, it sounds almost like it's a "self-test", like, a diagnostic tool or something, when it's actually an example showing you how Quarto docs can use R and Python. (I think this kinda clarity is important because the gallery does host both example content and extensions. 🙂)

The document name contributes to that impression, as does "exercises". Maybe just something like "Multi-Runtime Quarto Doc" or I dunno, "Runtime Combo Platter" if you want something fun (look it's Friday don't expect great wordsmithing from me lol). And maybe "A Quarto document that uses both R and Python" in the description?